db/migrate/20151229184253_create_token_auth_synchronizable_resources.rb

```diff
@@ -13,3 +13,13 @@ def change
     end
   end
 end
+
+if ActiveRecord::Migration.respond_to? :[]
+  class CreateTokenAuthSynchronizableResources < ActiveRecord::Migration[4.2]
```

Comment:
I didn't realize that the migrations won't work unless you specify.  I saw that the new docs use that notation, but I wasn't aware it was required.  Thanks.